conf/search_query_builder.conf

```diff
@@ -0,0 +1,18 @@
+# Display the query builder
+query_builder_enabled = 1
+
+# Global config for the query builder.  See http://querybuilder.js.org/#usage for available settings.
+query_builder_global_options = {
+    # Maximum nesting of groups
+    allow_groups = 4,
+    # No-value option for filter select box
+    select_placeholder = --- Select field for filter ---
+}
+
+# Per-table list of field/filter names to exclude from the query builder.
+query_builder_exclude_ca_objects = [
```

Comment:
This is a starting point; we should discuss with the client if there are specific fields they want to promote or exclude.